pages/common/ansible-inventory.md

```diff
@@ -0,0 +1,21 @@
+# ansible-inventory
+
+> Display or dump an Ansible inventory.
+> See also: `ansible`.
+> More information: <https://www.ansible.com/>.
+
+- Display the default inventory:
+
+`ansible-inventory --list`
+
+- Display a custom inventory:
+
+`ansbile-inventory --list --inventory {{path/to/inventory}}`
+
+- Display the default inventory in YAML:
+
+`ansible-inventory --list --yaml`
```

Comment:
Why not?
```suggestion
- Display the default inventory in YAML/TOML:

`ansible-inventory --list --{{yaml|toml}}`
```

Reply:
I thought of that but if the toml library isn't installed it gives an (informative) error message. yaml works out of the box.

Reply:
Hmm, then don't.